bootstrap/node.sh

```diff
@@ -29,4 +29,4 @@ modprobe dm_thin_pool
 
 echo "Try to join master..."
 # shellcheck disable=SC2154
-kubeadm join --token "${kubeadm_token}" "${master_ip}:6443"
+kubeadm join --discovery-token-unsafe-skip-ca-verification --token "${kubeadm_token}" "${master_ip}:6443"
```

Comment:
Please remove the `--discovery-token-unsafe-skip-ca-verification `

Reply:
As far as I understand: if we use `--discovery-token-unsafe-skip-ca-verification`, then we use the same as we did before,  the default in Kubernetes 1.7 and earlier. This mode relies only on the symmetric token to sign (HMAC-SHA256). Please consult https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-join/ for more information regarding this security update.

Reply:
Good.